benchmark/scenarios/read_request_load.py

```diff
@@ -0,0 +1,250 @@
+# Copyright 2015 ClusterHQ Inc.  See LICENSE file for details.
+"""
+Read request load scenario for the control service benchmarks.
+"""
+from collections import deque
+from itertools import repeat
+
+from zope.interface import implementer
+from eliot import start_action, write_failure, Message
+from eliot.twisted import DeferredContext
+
+from twisted.internet.defer import CancelledError, Deferred
+from twisted.internet.task import LoopingCall
+
+from flocker.common import loop_until, timeout
+
+from .._interfaces import IScenario
+
+DEFAULT_SAMPLE_SIZE = 5
+
+
+class RateMeasurer(object):
+    """
+    Measures the rate of requests in requests per second.
+
+    :ivar sample_size: The number of samples to collect.
```

Comment:
It appears that Twisted code documents internal attributes, but that the documentation processing strips out attributes that start with an underscore.  Hence, please add the private attributes to the docstring.